tests/unit/library.cairo

```diff
@@ -191,6 +198,8 @@ namespace it {
         let (local old_slot) = ERC3525.slot_of(from_token_id);
         %{
             stop_prank = start_prank(ids.caller) 
+            mock_call(ids.old_owner, "onERC3525Received", [ids.IERC3525_RECEIVER_ID])
+            mock_call(ids.old_owner, "supportsInterface", [1])
```

Comment:
to be consistent with the other test, do we want to change `[1]` to `[ids.TRUE]`  ?